src/Resources/ResourceManager/Implementation/ResourceGroups/ExportAzureResourceGroupCmdlet.cs

```diff
@@ -22,6 +22,9 @@ namespace Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation
     using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Entities.ResourceIds;
     using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Extensions;
     using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Utilities;
+    using Microsoft.Azure.Management.Internal.ResourceManager.Version2018_05_01.Models;
```

Comment:
Why did we have to import Models from internal ResourceManager SDK?

Reply:
Ah. That's not needed. I'll remove that